tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -659,7 +679,7 @@ public function jsonRespondedShouldContain(
 	}
 
 	/**
-	 * @Then /^the json responded should not contain a space "([^"]*)"$/
+	 * @Then /^the json responded should not contain a space with name "([^"]*)"$/
```

Comment:
Should here be also rename: https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/shareSpaces.feature#L64

Shouldn't there be error in CI? Test was passed
![image](https://user-images.githubusercontent.com/84779829/149356907-220511cf-798c-465d-b948-57ccffcabfb0.png)

Reply:
yes, i changed the step, didn't see that it was used in that other spot.